pkg/proxy/ipvs/proxier.go

```diff
@@ -650,11 +650,13 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 			if err != nil {
 				klog.Warningf("Failed to load kernel module %v with modprobe. "+
 					"You can ignore this message when kube-proxy is running inside container without mounting /lib/modules", module)
+			} else {
+				lmods = append(lmods, module)
 			}
 		}
 	}
 
-	return append(mods, bmods...), nil
+	return append(mods, append(bmods, lmods...)...), nil
```

Comment:
This is a bit hard to read. Maybe this is simpler: 

```
mods = append(mods, bmods...)
mods = append(mods, lmods...)

return mods, nil
```

Reply:
Addressed, thanks